examples/cpp/esp32-testbed/2_idf_build.sh

```
@@ -0,0 +1,7 @@
+clear
+
+export WORKLOAD_PRESET=esp32-s3-workloads-all
+export ROBOTICK_PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/.."
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)